### PR TITLE
Update unicycler to v0.4.8

### PR DESCRIPTION
**Pull request by dfornika**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/unicycler/unicycler.xml [hunk 1]
```diff
@@ -1,7 +1,16 @@
 <tool id="unicycler" name="Create assemblies with Unicycler" version="@VERSION@.0">
     <macros>
-        <token name="@VERSION@">0.4.7</token>
+        <token name="@VERSION@">0.4.8</token>
     </macros>
+    <edam_topics>
```
Comment (bgruening) [inline tools/unicycler/unicycler.xml]: Love that!!!

File: tools/unicycler/unicycler.xml [hunk 2]
```diff
@@ -8,9 +8,6 @@
     <edam_operations>
         <edam_operation>operation_0525</edam_operation>
     </edam_operations>
-    <xrefs>
```
Comment (bgruening) [inline tools/unicycler/unicycler.xml]: How urgent to you need this tool. xref are great and I think we just need a new planemo release to get linting pass.

Comment (dfornika) [reply tools/unicycler/unicycler.xml]: It's not urgent for me. I can add it back in to this PR, or I could submit another PR later to add the `<xrefs>` tag when the new planemo release has been deployed.

**Comment by bgruening** [comment]
Danke!

**Pull request merged**